docs/content/doc/installation/from-package.en-us.md

```diff
@@ -47,9 +47,52 @@ pacman -S gitea
 
 There is a [Gitea Snap](https://snapcraft.io/gitea) package which follows the latest stable version.
 
-``sh
-snap install gitea
-``
+### This guide was tested with Ubuntu 20.04 and MySQL:
+
+* Install needed Packages and the snap:
+
+```
+sudo snap install gitea
+sudo apt install nginx mariadb-server
+```
+* Secure your mysql Installation
+
+``sudo mysql_secure_installation``
```

Comment:
````suggestion
```bash
sudo mysql_secure_installation
```
````

Reply:
`apt install mariadb-server` has done such steps. This step shouldn't be included in this document.

Reply:
```suggestion
```

Reply:
> `apt install mariadb-server` has done such steps. This step shouldn't be included in this document.

I have to disagree (at least on Linux) if you don't run the script the root user for example has no password, root access is allowed from external sources etc.

https://mariadb.com/kb/en/installing-mariadb-deb-files/#installing-mariadb-packages-with-apt
https://mariadb.com/kb/en/mysql_secure_installation/